stdlib/public/Concurrency/Task.swift

```diff
@@ -481,6 +481,55 @@ public func detach<T>(
   return Task.Handle<T, Error>(task)
 }
 
+/// Run given `operation` as asynchronously in its own top-level task.
+///
+/// The `async` function should be used when creating asynchronous work
+/// that operations on behalf of the synchronous function that calls it.
+/// Like `detach`, the async function creates a separate, top-level task.
+/// Unlike `detach`, the task creating by `async` inherits the priority and
+/// actor context of the caller, so the `operation` is treated more like an
+/// asynchronous extension to the synchronous operation. Additionally, `async`
+/// does not return a handle to refer to the task.
+///
+/// - Parameters:
+///   - priority: priority of the task. If unspecified, the priority will
+///               be inherited from the task that is currently executing
+///               or, if there is none, from the platform's understanding of
+///               which thread is executing.
+///   - operation: the operation to execute
+@available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
+public func async(
```

Comment:
Man... I'd really like a verb here; Isn't this a `send`?

- spawn - make child task, will await it
- detach - completely separate; could wait if one wanted to, carries no values/things implicitly
- send - fire and forget, cannot await on the result; usable from sync code, does carry priority and locals <- this is exactly what doing a "fire and forget" message send would want to be, so I'm so tempted to call it a send... `async` is a bit weird and it's dispatch wording inheritance too..

Reply:
> Man... I'd really like a verb here; Isn't this a `send`?

Nah, it's a `dispatch` ;-)
(yeah, gcd connotations I know, but one could say the same for `send(2)`)